src/main/java/seedu/address/MainApp.java

```diff
@@ -84,6 +87,8 @@ public void init() throws Exception {
     private Model initModelManager(Storage storage, UserPrefs userPrefs) {
         Optional<ReadOnlyAddressBook> addressBookOptional;
         ReadOnlyAddressBook initialData;
+        //Optional<ReadOnlyTranscript> transcriptOptional;
+        //ReadOnlyTranscript initialTranscriptData;
```

Comment:
Is this also part of `Not used for now` like the later part?

Reply:
Yes.